### ISSUE TITLE
Tool doesn't work reliably with Win10 x64

### ISSUE THREAD

**Issue by OevreFlataeker** [mrexodia/ArmaG3ddon#1]
The dumper does not work well with Win10 x64. Sometimes one has to repeat the dump activity several times until the option to save the dump is offered. Sometimes there are errors in ReadProcessMemory or the application just launches but Armag3ddon does not detect this and does not offer to dump it.

Any chance to make the tool compatible with Win10 x64? The original source as mentioned on the homepage is 404. Is there a newer source known?

**Comment by adoxa** [comment]
A quick test worked fine for me on Win10 x64 (under Sandboxie). Got a link to something I can test? (Although no promises when.)

This was the most recent at the time I updated; no activity on the forks, so apparently there is nothing newer.

**Comment by OevreFlataeker** [comment]
Yeah I have some things to try but don't wanna post here. Any way of privately transmitting that information to you?
